examples/interactive_play.py

```diff
@@ -310,8 +308,14 @@ def get_input_vel_ctlr(
         else:
             # Use the controller
             relative_pos = mn.Vector3(base_action[0], 0, base_action[1])
-            pose, root_trans = humanoid_controller.get_walk_pose(relative_pos)
-            base_action = humanoid_controller.vectorize_pose(pose, root_trans)
+            (
+                pose,
+                root_trans_offset,
+                root_trans_base,
+            ) = humanoid_controller.get_walk_pose(relative_pos)
+            base_action = humanoid_controller.vectorize_pose(
+                pose, root_trans_offset, root_trans_base
+            )
```

Comment:
A tiny bit awkward that the 3 outputs of get_walk_pose are passed to vectorize_pose. Would it be simpler to have a humanoid_controller.action_from_relative_position that takes as input `relative_pos` and returns `base_action` (under the hood it would just call get_walk_pose then vectorize_pose.
If this is the only place this is used, maybe doing 
```
full_pose = humanoid_controller.get_walk_pose(relative_pos)
base_action = humanoid_controller.vectorize_pose(* full_pose)
```

Reply:
Yes, I like that. I would like to keep the original functions that return the different components, in case we want to use part of the transform lately (for instance, compute the joint changes but not the base transform), but I can have a function that calls + vectorize under the hood.